### DISCUSSION TITLE
post-it (beta)

### DISCUSSION THREAD

**Discussion by berot3** [chrisgrieser/shimmering-focus#174] [Q&A] [answered]
I really like how post-it-callouts are residing on the side and would like to use it more. But since they are not visible at all in the Live Preview, it is a bit hard to work with. (they simply disappear for me)

I think it would be great to make them visible in Live Preview (I understand that they cannot move to the side like in Reading mode).

And maybe give them a yellow background in Live Preview so they are even more visible?

**Comment by chrisgrieser** [comment, marked as answer]
Sorry for the late response!

Due to me having to rewrite the whole theme for Obsidian 0.16, the post-it feature is unfortunately one of the features that (for now) didn't make the cut yet.

If after the release of Obsidian 0.16 you are still interested in the feature, please open a feature request 🙂
